### PR TITLE
[travis] Fix deps=high/low jobs

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #15230 |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,7 @@
...
-autoload.php
+vendor/
 composer.lock
-composer.phar
-package*.tar
-packages.json
 phpunit.xml
-/vendor/
+.php_cs.cache
+composer.phar
```
Comment (stof) [inline .gitignore]: why changing the order in this file ?

Comment (nicolas-grekas) [inline .gitignore]: to make it more inline with the other ones

File: .travis.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    $package->version = $branch.'.x-dev';
+    $package->dist['type'] = 'tar';
+    $package->dist['url'] = 'file://'.__DIR__."/$dir/package.tar";
+
+    $packages[$name][$package->version] = $package;
+
+    $package = file_get_contents('https://packagist.org/packages/'.$name.'.json');
+    $package = json_decode($package);
+
+    foreach ($package->package->versions as $version => $package) {
```
Comment (stof) [inline .travis.php]: you should not overwrite the ``$package`` variable here IMO. It hurts readability

Comment (nicolas-grekas) [inline .travis.php]: fixed

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**